### PR TITLE
Mohit chawla/template and file upload

### PR DESCRIPTION
**Pull request by mohit-chawla**
# Description 
Adds file upload feature using dropzone

Please include a summary of the change and which issue is fixed. Please also include related PRs. List any dependencies that are required for this change.

# Test/Benchmark Outputs

# Checklist:
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Related Benchmarks have been performed, if applicable.
- [x] I have a plan to revert my changes.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if applicable.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
```
Comment (kriticism) [inline .gitignore]: Can you please check and see why we have this line again when a previous PR had already added this line to master?

File: board/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                <meta http-equiv="Content-Language" value="en-US" />
+            {% endblock head_meta_contentlanguage %}
+            {% block head_meta_viewport %}
+                <meta name="viewport"
+                      content="width=device-width, initial-scale=1.0">
+            {% endblock head_meta_viewport %}
+        {% endblock head_meta %}
+        {% block head_css %}
+            {% block head_css_site %}
+               <!--  <link href="{{ STATIC_URL }}css/bootstrap.min.css"
```
[the remark this answers is not in the document]
Comment (mohit-chawla) [reply board/templates/base.html]: We can, but we should not, i plan to uncomment this later when we bootstrap the whole thing

File: board/models.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
 from django.db import models
-
-# Create your models here.
+ 
+#  #WARNING: NOT USED AS OF NOW
```
Comment (kriticism) [inline board/models.py]: Is this comment needed? if yes, pl give a one liner, else remove.

Comment (mohit-chawla) [reply board/models.py]: ideally we should upload file related data in db. This comment says that we should do this later. (After we finalize db and its schema)

File: .gitignore (hunk 1 above)
Comment (mohit-chawla) [reply .gitignore]: fixed

File: board/static/css/dropzone.css [hunk 4]
```diff
@@ -0,0 +1,388 @@
+/*
+ * The MIT License
+ * Copyright (c) 2012 Matias Meno <m@tias.me>
+ */
+@-webkit-keyframes passing-through {
```
[the remark this answers is not in the document]
Comment (mohit-chawla) [reply board/static/css/dropzone.css]: Lets avoid cdns for now

**Pull request merged**